sotodlib/hwp/hwp.py

```diff
@@ -11,7 +11,7 @@ def get_hwpss(aman, signal=None, hwp_angle=None, bin_signal=True, bins=360,
               lin_reg=True, modes=[1, 2, 3, 4, 6, 8], apply_prefilt=True,
               prefilt_cfg=None, prefilt_detrend='linear', flags=None,
               merge_stats=True, hwpss_stats_name='hwpss_stats',
-              merge_model=True, hwpss_model_name='hwpss_model'):
+              merge_model=True, hwpss_model_name='hwpss_model', **kwargs):
```

Comment:
why do you need to add **kwargs here if there's no changes later?

Reply:
I needed to add this since these functions don't take `save_plot` which is within `**self.calc_cfgs` that's being passed